package.json

```diff
@@ -28,7 +28,7 @@
     "@material-ui/icons": "^4.11.2",
     "@material-ui/lab": "^4.0.0-alpha.58",
     "@material-ui/styles": "^4.11.4",
-    "@saleor/macaw-ui": "0.3.1",
+    "@saleor/macaw-ui": "github:saleor/macaw-ui#SALEOR-5995-switch-placement",
```

Comment:
Make sure to release macaw before merging this in (the branch will be deleted after it's merged and build will fail)

Reply:
Maybe we should introduce a CI check to make sure we don't use GH packages inside package.json